gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -2364,14 +2365,57 @@ else if (value instanceof Traversal)
      * @since 3.0.0-incubating
      */
     public default GraphTraversal<S, E> property(final Object key, final Object value, final Object... keyValues) {
```

Comment:
could you please have a look at the javadoc for both of these `property()` methods - they likely need updating given all this change, right?